modules/feature-foo/src/main/kotlin/com/leinardi/forlago/feature/foo/ui/foo/FooContract.kt

```diff
@@ -16,20 +16,22 @@
 
 package com.leinardi.forlago.feature.foo.ui.foo
 
+import androidx.compose.runtime.Immutable
 import com.leinardi.forlago.library.ui.base.ViewEffect
 import com.leinardi.forlago.library.ui.base.ViewEvent
 import com.leinardi.forlago.library.ui.base.ViewState
 
+@Immutable
 object FooContract {
     data class State(
         val text: String,
-        val isLoading: Boolean = false,
+        val loading: Boolean = false,
```

Comment:
Alphabetical order

Reply:
Properties with default values go after the ones without default value.